packages/fiber/src/core/store.ts

```diff
@@ -1,6 +1,5 @@
 import * as THREE from 'three'
-import * as React from 'react'
-import create, { GetState, SetState, StoreApi, UseBoundStore } from 'zustand'
+import create, { GetState, SetState, StoreApi } from 'zustand/vanilla'
```

Comment:
Using vanilla store for `core` and is consumed with `create` from `zustand` in `react/index.tsx`::`createRoot`